src/scripts/buttons.js

```diff
@@ -0,0 +1,70 @@
+function getPathObj(buttonPath, buttonText=null, buttonDescription=null) {
```

Comment:
This file can be refactored to a much smaller script operating on normal HTML with data attributes automatically, I'm 99% certain.